Remotion/ObjectBinding/Web.IntegrationTests/BocTreeView/BocTreeViewTest.cs

```diff
@@ -48,6 +48,25 @@ public void WebTreeView_ExpansionOfEmptyEmitsNoItemFoundText ()
     Assert.That(() => ariaTreeAlertSpan.GetAttribute("aria-hidden"), Is.EqualTo("true").After(7).Seconds.PollEvery(200).MilliSeconds);
   }
 
+  [Test]
+  public void WebTreeView_Selected_HasAriaSelectedAttributeTrue ()
```

Comment:
Since the change is static code rendered by the server, I would suggest not using a webtest to verify this. I know we don't test the rendering for TreeView any other way, but I would still dropp the test due to the runtime effort involved and setting a precent (i.e. you suddenly find webtests for stuff that should be tested as a unit test and are unsure when to use what). In theory, we could start building unit tests for WebTreeView rendering but the usual disclaimer applies: is this change big enough to warrent the expense?

Reply:
If you want to keep the test, I would recommend adding a comment that this is only since we don't have a unit test for it.